patches/ui-base-BUILD.gn.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
-index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..25349fd0787e0fd9c913fab16d3c646bf856f1f9 100644
+index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..3fbf1a1e7276481638ad79943b9e4cfa51ed6698 100644
```

Comment:
@mkarolin is this what you're saying is not needed? Besides the below `deps += [ "//brave/ui/base:chromium_deps" ]`, this should be the only diff in this patch